lib/tidewave/mcp/tools/source.ex

```diff
@@ -157,4 +168,91 @@ defmodule Tidewave.MCP.Tools.Source do
 
     Path.join([lib_or_src | Enum.reverse(in_app)])
   end
+
+  # IEx.Introspection.decompose
+
+  defp decompose(atom, _context) when is_atom(atom), do: atom
+
+  defp decompose({:__aliases__, _, _} = module, context) do
+    Macro.expand(module, context)
+  end
+
+  defp decompose({:/, _, [call, arity]}, context) do
+    case Macro.decompose_call(call) do
+      {_mod, :__info__, []} when arity == 1 ->
+        {Module, :__info__, 1}
+
+      {mod, fun, []} ->
+        {Macro.expand(mod, context), fun, arity}
```

Comment:
We don't need to expand it either.

Reply:
if we don't we get this is mod, example:

```
{:__aliases__, [line: 1], [:Tidewave, :MCP, :Tools, :Source]}
```